<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteSettingsWidgetAnyCommand</class>
 <widget class="QWidget" name="RemoteSettingsWidgetAnyCommand">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>375</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="downloadGroupBox">
     <property name="title">
      <string>Download</string>
     </property>
     <layout class="QFormLayout" name="downloadFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="downloadCommandLabel">
        <property name="text">
         <string>Command:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="downloadCommand">
        <property name="accessibleName">
         <string>Download command field</string>
        </property>
        <property name="placeholderText">
         <string>e.g.: sftp user@hostname</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="downloadCommandInputLabel">
        <property name="text">
         <string>Input:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="inputForDownload">
        <property name="accessibleName">
         <string>Download input field</string>
        </property>
        <property name="placeholderText">
         <string>e.g.:
get DatabaseOnRemote.kdbx {TEMP_DATABASE}
exit
---
{TEMP_DATABASE} is used as placeholder to store the database in a temporary location
The command has to exit. In case of `sftp` as last commend `exit` has to be sent</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="uploadGroupBox">
     <property name="title">
      <string>Upload</string>
     </property>
     <layout class="QFormLayout" name="uploadFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="uploadCommandLabel">
        <property name="text">
         <string>Command:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="uploadCommand">
        <property name="accessibleName">
         <string>Upload command field</string>
        </property>
        <property name="placeholderText">
         <string>e.g.: sftp user@hostname</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="uploadCommandInputLabel">
        <property name="text">
         <string>Input:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="inputForUpload">
        <property name="accessibleName">
         <string>Upload input field</string>
        </property>
        <property name="placeholderText">
         <string>e.g.:
put {TEMP_DATABASE} DatabaseOnRemote.kdbx
exit
---
{TEMP_DATABASE} is used as placeholder to store the database in a temporary location
The command has to exit. In case of `sftp` as last commend `exit` has to be sent</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
